### PR TITLE
Hulya - Create the header component

### LINKED ISSUE #40: As a user, I want to have my token displayed for me somewhere so that I don't have to search for it
Display the user's token in the header when they sign into their list so they don't have to go into `localStorage` and find it.

Included in Header Component issue #44

### LINKED ISSUE #44: Header Component

### PR DESCRIPTION
**Pull request by hulyak**
## Description

- Created Header component displaying Header logo and "Lettuce Know" title.
- Displayed the user's token under the logo.
- Header component is visible on Add Item and List page.
- Incorporated responsive font size for the Header text with media queries inside the `src/themes/theme.js` file
- Used Material-UI Box, and Typography components.

I had an issue with the local SVG file, that's why passed the URL for the image source. 

## Related Issue

closes #44 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    | :sparkles: New feature     |
|  ✓| :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

Screen Shot 2021-08-31 at 4 23 13 PM
Screen Shot 2021-08-31 at 4 24 02 PM

### After

Screen Shot 2021-08-31 at 4 16 03 PM
Screen Shot 2021-08-31 at 4 16 10 PM
Screen Shot 2021-08-31 at 4 16 18 PM

## Testing Steps / QA Criteria

-  `git checkout hk-header`
- `npm start`
- Check the pages to see the header component

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <>
+      <div style={{ width: '100%', marginTop: '2em' }}>
+        <Box display="flex" alignItems="center" justifyContent="center" p={1}>
+          <img
+            src="https://image.flaticon.com/icons/png/512/1000/1000145.png"
+            alt="lettuce icon"
+            height="60"
+          />
+          <Typography variant="h2">Lettuce Know</Typography>
+        </Box>
```
Comment (sandaiiyahh) [inline src/components/Header.js:16]: Cool incorporation of the Box component! Never heard of it before

**Review by sandaiiyahh — APPROVED** [review]
Awesome job, Hulya! The header looks amazing~I love how you added media queries too! 💯

**Review by ksiman14 — APPROVED** [review]
I love the lettuce icon you chose @hulyak, it's so cute!

**Comment by isaabutaa** [comment]
Great job on the header component, @hulyak! It looks great, and I love the responsiveness. 💯

**Review by isaabutaa — APPROVED** [review]

**Pull request merged**